### ISSUE TITLE
RealGANModel模式各个损失的loss_weight设置

### ISSUE THREAD

**Issue by erxianzhe** [sunny2109/SAFMN#34]
在RealGANModel模式下，各个损失的loss_weight应该如何设置比较合理

**Comment by sunny2109** [comment]
> 在RealGANModel模式下，各个损失的loss_weight应该如何设置比较合理

不好意思，回复晚了。
训练的时候，我用到了这些loss: L1, FFT Loss, Perceptual Loss, LDL Loss, GAN Loss
Stage 1: L1 + 0.1\*FFT Loss;  Stage2: L1 + 0.1\*GAN Loss + Perceptual Loss;  Stage 3: L1 + 0.1\*GAN Loss + Perceptual Loss + 0.5\*LDL Loss

**Comment by harusaju** [comment]
> In RealGANModel mode, how should the loss_weight of each loss be set more reasonably?
Hi, did you manage to train the model in RealGANModel mode?

**Comment by sunny2109** [comment]
> > In RealGANModel mode, how should the loss_weight of each loss be set more reasonably?
> 
> Hi, did you manage to train the model in RealGANModel mode?

Yep, the degradation settings are almost identical to RealESRGAN.
